windows/src/desktop/kmshell/main/Keyman.System.UpdateStateMachine.pas

```diff
@@ -630,10 +639,14 @@ procedure UpdateAvailableState.StartDownloadProcess;
 begin
   // call seperate process
   RootPath := ExtractFilePath(ParamStr(0));
-  FResult := TUtilExecute.ShellCurrentUser(0, ParamStr(0), IncludeTrailingPathDelimiter(RootPath), '-bd');
+  FResult := TUtilExecute.ShellCurrentUser(0, ParamStr(0),
+    IncludeTrailingPathDelimiter(RootPath), '-bd');
   if not FResult then
-    // TODO: #10210 Log to Sentry
-    KL.Log('TrmfMain: Executing KMshell for download updated Failed');
+  begin
+    TKeymanSentryClient.Client.MessageEvent(Sentry.Client.SENTRY_LEVEL_ERROR,
+      'Executing kmshell process to download updated Failed');
```

Comment:
I guess we don't get an error code or something that we could output here to get a better idea what went wrong?